react-loops.js.flow

```diff
@@ -28,3 +28,6 @@ declare function For<O: Object>({|
 declare function If({ case: any, children: React$Node }): React$Node;
 declare function ElseIf({ case: any, children: React$Node }): React$Node;
 declare function Else({ children: React$Node }): React$Node;
+declare function Choose({ children: React$Node<typeof When> | React$Node<typeof Otherwise>}): React$Node;
```

Comment:
Does this line work in a try flow?

I'd have expected `React$Node<When | Otherwise>`